test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -888,11 +888,12 @@
 (defmethod mt/get-dataset-definition TimestampDatasetDef
   [^TimestampDatasetDef this]
   (let [interval-seconds (.intervalSeconds this)]
-    (mt/dataset-definition (str "checkins_interval_" interval-seconds)
-      ["checkins"
-       [{:field-name "timestamp"
-         :base-type  (or (driver->current-datetime-base-type driver/*driver*) :type/DateTime)}]
-       (vec (for [i (range -15 15)]
+    (mt/dataset-definition
+     (str "checkins_interval_" interval-seconds)
+     ["checkins"
+      [{:field-name "timestamp"
+        :base-type  (or (driver->current-datetime-base-type driver/*driver*) :type/DateTime)}]
+      (mapv (fn [i]
```

Comment:
I reworked this to use `mapv` so it's not lazy, it's important because it needs to be done while `hx/*honey-sql-version*` is bound. Also makes it easier to debug

Reply:
I don't think so. Things will just cause tests to fail if we're returning lazy sequences and we can fix places where that happens as they come up. I don't anticipate there being many more than what we've already fixed.